src/components/ui/Merge/Memes/MergeProcessingModal.tsx

```
@@ -0,0 +1,117 @@
+    hash: hashApproveToken,
+    isPending: isPendingApproveToken,
+    isConfirming: isConfirmingApproveToken,
+    isError: isErrorApproveToken,
+    approve: approveToken,
+  } = useApproveToken();
+
+  const {
+    hash: hashEatToken,
+    isPending: isPendingEatToken,
+    isError: isErrorEatToken,
+    eat: eat,
```

Biome

[error] Useless rename.

Safe fix: Remove the renaming.


(lint/complexity/noUselessRename)